__tests__/app/dashboard/user/create/page.unit.test.tsx

```
@@ -1,354 +1,206 @@
+  fireEvent.change(screen.getByLabelText(/Email/i), { target: { value: "johndoe@example.com" } });
+  fireEvent.change(screen.getByLabelText(/Password/i), { target: { value: "password123" } });
+  fireEvent.click(screen.getByText("Pilih Divisi"));
+  fireEvent.click(screen.getByRole("option", { name: "IT Division" }));
+  fireEvent.click(screen.getByText("Pilih Role"));
+  fireEvent.click(screen.getByRole("option", { name: "Admin" }));
+  fireEvent.click(screen.getByText("Pilih tanggal"));
+  fireEvent.click(screen.getByText("15")); // Select a date from the calendar
+};
+
+// Helper function to mock fetch behavior
+const mockFetch = (response: any, ok = true) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)